internal/mget.go

```diff
@@ -0,0 +1,14 @@
+package internal
+
+import (
+	"context"
+	"github.com/CrowdStrike/opensearchtools"
+)
+
+// MGet defines a method which knows how to make an OpenSearch multiple document get.
+// It should be implemented by a version-specific executor.
+type MGet[K opensearchtools.RoutableDoc] interface {
```

Comment:
Does this need to be generic now?  Nothing is using `K`.

Reply:
Good call